.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      matrix:
```

Comment:
Should Java version be a part of the configuration too? I would suggest 11, 17 and 21 for now

Reply:
450f902e4d6125128b2f13978a4e3243a5ae5d15